src/doc-mirror/index.ts

```diff
@@ -169,18 +169,21 @@ export class MirroredDocument implements EditableDocument {
   }
 
   public delete(): Thenable<boolean> {
-    return vscode.commands.executeCommand('deleteRight');
+    return vscode.commands.executeCommand('deleteRight').then((v) => !!v);
```

Comment:
The actual return type here was `Thenable<true | undefined>` and so this fixes the type so it's `Thenable<boolean>` again.